examples/example1a_SAM/example1a.py

```diff
@@ -9,13 +9,13 @@
 
 # TH params
 
-params['He_inlet_temp'] = 600 + 273.15  # K
-params['He_outlet_temp'] = 850 + 273.15 # K
-params['He_cp'] = 5189.2 # J/kg-K
+params['He_inlet_temp'] = {"value":600, "current_unit": "Celsius"}  # 873.15 K
```

Comment:
I'm still not a fan of providing this as a dictionary, since we already allow that for other purposes. I would suggest just setting this using the `Quantity` class directly:
```Python
from astropy.units import Quantity
...

params['He_inlet_temp'] = Quantity(600, "Celsius")
```